src/Pages/ProductsPage/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
You names the folder `ProductsPage` but you are in `Pages` folder so it's redundant,
please change folder name to just `Products`

Reply:
fixed